### DISCUSSION TITLE
onBackPressedDispatcher

### DISCUSSION THREAD

**Discussion by Arvant** [axmolengine/axmol#1318] [General]
now in android onbackpress is deprecated and view need to use onBackPressedDispatcher on android how i can add it to axmol project

**Comment by halx99** [comment]
do you mean: `ax::EventKeyboard::KeyCode::KEY_ESCAPE`: https://github.com/axmolengine/axmol/blob/dev/core/platform/android/jni/TouchesJni.cpp#L105

**Reply by Arvant** [reply]
no. please see this link that is about it. https://developer.android.com/guide/navigation/custom-back/predictive-back-gesture
